### PR TITLE
Fix wording per Stefan's feedback

### PR DESCRIPTION
**Pull request by dooriya**
Fix bug: https://dev.azure.com/mseng/VSIoT/_workitems/edit/1584336

### CODE REVIEW

File: resources/templates/ansic_cmake_connectionstring/Readme.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-For more details about setting up development environment for compiling the C Device SDK. Check the [instructions](https://github.com/Azure/azure-iot-sdk-c/blob/master/iothub_client/readme.md#compiling-the-c-device-sdk) for each platform.
+For more details about setting up your development environment for compiling the C Device SDK. Check the [instructions](https://github.com/Azure/azure-iot-sdk-c/blob/master/iothub_client/readme.md#compiling-the-c-device-sdk) for each platform.
 
 ## Windows
 
-1. Install [Visual Studio 2017](https://www.visualstudio.com/downloads/). You can use the Visual Studio Community Free download if you meet the licensing requirements. (Visual Studio 2015 is also supported.)
+1. Install [Visual Studio](https://www.visualstudio.com/downloads/). You can use the Visual Studio Community Free download if you meet the licensing requirements. (Visual Studio 2015 is also supported.)
 
-    > Be sure to include Visual C++ and NuGet Package Manager.
+    > Be sure to include **Desktop development with C++** and **NuGet Package Manager** from the Visual Studio Installer.
```
Comment (liydu) [inline resources/templates/ansic_cmake_connectionstring/Readme.md]: and **NuGet Package Manager** workloads from...

File: resources/templates/ansic_cmake_iotcsaskey/Readme.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
-For more details about setting up development environment for compiling the C Device SDK. Check the [instructions](https://github.com/Azure/azure-iot-sdk-c/blob/master/iothub_client/readme.md#compiling-the-c-device-sdk) for each platform.
+For more details about setting up your development environment for compiling the C Device SDK. Check the [instructions](https://github.com/Azure/azure-iot-sdk-c/blob/master/iothub_client/readme.md#compiling-the-c-device-sdk) for each platform.
 
 ## Windows
 
-1. Install [Visual Studio 2017](https://www.visualstudio.com/downloads/). You can use the Visual Studio Community Free download if you meet the licensing requirements. (Visual Studio 2015 is also supported.)
+1. Install [Visual Studio](https://www.visualstudio.com/downloads/). You can use the Visual Studio Community Free download if you meet the licensing requirements. (Visual Studio 2015 is also supported.)
 
-    > Be sure to include Visual C++ and NuGet Package Manager.
+    > Be sure to include **Desktop development with C++** and **NuGet Package Manager** from the Visual Studio Installer.
```
Comment (liydu) [inline resources/templates/ansic_cmake_iotcsaskey/Readme.md]: and **NuGet Package Manager** workloads from...

File: resources/templates/ansic_vs_connectionstring/Readme.md [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
-For more details about setting up development environment for compiling the C Device SDK. Check the [instructions](https://github.com/Azure/azure-iot-sdk-c/blob/master/iothub_client/readme.md#compiling-the-c-device-sdk) for each platform.
+For more details about setting up your development environment for compiling the C Device SDK. Check the [instructions](https://github.com/Azure/azure-iot-sdk-c/blob/master/iothub_client/readme.md#compiling-the-c-device-sdk) for each platform.
 
 ## Windows
 
-1. Install [Visual Studio 2017](https://www.visualstudio.com/downloads/). You can use the Visual Studio Community Free download if you meet the licensing requirements. (Visual Studio 2015 is also supported.)
+1. Install [Visual Studio](https://www.visualstudio.com/downloads/). You can use the Visual Studio Community Free download if you meet the licensing requirements. (Visual Studio 2015 is also supported.)
 
-    > Be sure to include Visual C++ and NuGet Package Manager.
+    > Be sure to include **Desktop development with C++** and **NuGet Package Manager** from the Visual Studio Installer.
```
Comment (liydu) [inline resources/templates/ansic_vs_connectionstring/Readme.md]: Same as above.

File: resources/templates/ansic_vs_iotcsaskey/Readme.md [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
-For more details about setting up development environment for compiling the C Device SDK. Check the [instructions](https://github.com/Azure/azure-iot-sdk-c/blob/master/iothub_client/readme.md#compiling-the-c-device-sdk) for each platform.
+For more details about setting up your development environment for compiling the C Device SDK. Check the [instructions](https://github.com/Azure/azure-iot-sdk-c/blob/master/iothub_client/readme.md#compiling-the-c-device-sdk) for each platform.
 
 ## Windows
 
-1. Install [Visual Studio 2017](https://www.visualstudio.com/downloads/). You can use the Visual Studio Community Free download if you meet the licensing requirements. (Visual Studio 2015 is also supported.)
+1. Install [Visual Studio](https://www.visualstudio.com/downloads/). You can use the Visual Studio Community Free download if you meet the licensing requirements. (Visual Studio 2015 is also supported.)
 
-    > Be sure to include Visual C++ and NuGet Package Manager.
+    > Be sure to include **Desktop development with C++** and **NuGet Package Manager** from the Visual Studio Installer.
```
Comment (liydu) [inline resources/templates/ansic_vs_iotcsaskey/Readme.md]: Same.

File: resources/templates/ansic_cmake_connectionstring/Readme.md (hunk 1 above)
Comment (dooriya) [reply resources/templates/ansic_cmake_connectionstring/Readme.md]: NuGet Package Manager doesn't belongs to workloads, it is one of the Individual component

**Pull request merged**